gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -340,6 +342,7 @@ class GateConfig extends RedisHttpSessionConfiguration {
       .enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_AS_NULL)
       .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
       .registerModule(new JavaTimeModule())
+      .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
```

Comment:
Could you elaborate on why these changes are necessary?

Reply:
yeah with out setting this, when sending CDEvent POST request to this webhook will result in below error in the response;

`
{"error":"Internal Server Error","exception":"retrofit.RetrofitError","message":"com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class io.cloudevents.core.data.BytesCloudEventData and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: io.cloudevents.core.v1.CloudEventV1[\"data\"])"}
`

Reply:
Hm, I'd like to get @link108 or @dbyron-sf's thoughts on this. I'm hesitant to change this for the global `ObjectMapper` without thorough understanding of any side effects.

Reply:
This setting is what I need it for `echoservice`. 
If this is fine, I can only set it for `echo` as below also works for me
```suggestion
   if(serviceName.equals("echo")){
      objectMapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
   }
```

```
  
```

Reply:
@link108 @dbyron-sf any suggestion on this change?

Reply:
I'm not super familiar with the consequences of setting fail-on-empty-beans to false.  Maybe @jvz knows?

What would it take with the latest code on master to configure it that way only for echo?

Reply:
I'm not really sure myself what that does. I learn something new about Jackson every time I use it, though! If the setting isn't safe to set globally, then what you can do is update the `EchoService` bean to use the `objectMapperBuilder` field to create a custom `ObjectMapper` as that bean is prototype-scoped.

Reply:
Using the same way, setting this configuration only for echo as suggested above with in the `buildService` method